tangelo/linq/target/target_stim.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2021 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+
+import numpy as np
+from collections import Counter
+
+import stim
+from tangelo.linq import Circuit
+from tangelo.linq.target.backend import Backend
+from tangelo.linq.translator import translate_circuit as translate_c
+from tangelo.linq.translator.translate_stim import direct_tableau
+
+
+class StimSimulator(Backend):
+    """Interface to the stim simulator."""
+
+    def __init__(self, n_shots=None, noise_model=None):
+        import stim
+        from stim import TableauSimulator
```

Comment:
I don't think this helps us, and the end of the function `TableauSimulator` just vanishes and will need to be reimported later. You don't seem to use it here